src/SpectatorView.Unity/Assets/SpectatorView/Scripts/StateSynchronization/NetworkManager.cs

```diff
@@ -2,35 +2,36 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using System;
+using System.Collections.Generic;
 using System.IO;
 using UnityEngine;
 
 namespace Microsoft.MixedReality.SpectatorView
 {
     public abstract class NetworkManager<TService> : CommandRegistry<TService>, INetworkManager where TService : Singleton<TService>
     {
-        private float lastReceivedUpdate;
-
+#pragma warning disable 414 // The field is assigned but its value is never used

```

Comment:
It seems that this _is_ used below. What's the deal?

Reply:
stale code, removing now.